tests_integ/models/conformance.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from strands.types.models import Model
+from tests_integ.models.providers import ProviderInfo, all_providers
+
+
+def get_models():
+    return [
+        pytest.param(
+            provider_info,
+            id=provider_info.id,  # Adds the provider name to the test name
+            marks=[provider_info.mark],  # ignores tests that don't have the requirements
+        )
+        for provider_info in all_providers
+    ]
+
+
+@pytest.fixture(params=get_models())
+def provider_info(request) -> ProviderInfo:
+    return request.param
+
+
+@pytest.fixture()
+def model(provider_info):
+    return provider_info.create_model()
+
+
+def test_model_can_be_constructed(model: Model):
```

Comment:
This is an example of a test that runs against all providers.  We could continue adding to this file to make sure we have consistent tests across all providers and/or when testing new changes to check the behavior of different providers.

<img width="469" height="329" alt="Image" src="https://github.com/user-attachments/assets/58f3a5b2-40fd-469e-baab-4e087ee6ad35" />